.eslintrc.json

```
@@ -28,6 +52,10 @@
       }
     ],
     /* Rules to make it not necessary to include React as it is done automatically since React 17 */
-    "react/react-in-jsx-scope": "off"
+    "react/react-in-jsx-scope": "off",
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      { "argsIgnorePattern": "^_", "varsIgnorePattern": "^_" }
```

In a lot of places, we use object destructuring to grab some subset of keys from an object e.g.
```js
const {a, b, ...rest} = obj
// do something with rest
```
I think it makes sense to ignore the error in that case?
```suggestion
      { "argsIgnorePattern": "^_", "varsIgnorePattern": "^_", "ignoreRestSiblings": true } 
```

---

Makes sense, I agree